tools/prepare_python_system_tests.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import json
+import os
+import subprocess
+import tempfile
+import time
+import urllib.request
+import zipfile
+
+parser = argparse.ArgumentParser(description='Downloads the latest release nimi-python and runs system tests on the specified driver.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
"release of" or "released".

Reply:
Changed to `'Downloads the latest release atrifacts from nimi-python and runs system tests on the specified python package.'`